app.js

```diff
@@ -8,4 +58,86 @@ hamburguer.addEventListener('click', () => {
 navMenu.addEventListener('click', () => {
   hamburguer.classList.toggle('active');
   navMenu.classList.toggle('active');
-});
\ No newline at end of file
+});
+
+projects.forEach((work) => {
+  works.innerHTML += `
+    <div class="section-${work.type} main_section">
+                <div class="content_image_card">
+                    <img class="image_card" src="${work.image}" alt="snapshot tonic">
+                </div>
+                <div class="content_card">
+                    <h2 class="title_card">${work.title}</h2>
+                    <div class="subtitle_card">
+                        <h6 class="first_subtitle_card">${work.subtitle1}</h6>
+                        <img class="point_icon" src="images/point_icon.png" alt="point">
+                        <h6 class="second_subtitle_card">${work.subtitle2}</h6>
+                        <img class="point_icon" src="images/point_icon.png" alt="point">
+                        <h6 class="second_subtitle_card">${work.year}</h6>
+                    </div>
+                    <p class="paragraph_card">
+                        ${work.paragraph}
+                    </p>
+                    <div>
+                        <ul class="technologies">
+                            <li>${work.technologies[0]}</li>
+                            <li>${work.technologies[1]}</li>
+                            <li>${work.technologies[2]}</li>
+                        </ul>
```

Comment:
- [ ] Please you can use the map function in JavaScript to populate the `<ul>` element with the technologies from an array. Doing this will make your code simple and clean. Assume you have 5 technologies for `project1` and 3 technologies for `project2`, you won't be able to display precisely the 5 technologies for `project1` and the 3 technologies for the `project2` since your code will throw an issue at `${work.technologies[4]}` for `project2`. :blush:  But following the code below will help you add any number of technologies for each project :+1: 
  ```js
  const technologiesList = document.querySelector(".technologies");
  const technologiesItems = work.technologies.map(technology => `<li>${technology}</li>`);
  technologiesList.innerHTML = technologiesItems.join("");
  ```